README.md

```diff
@@ -150,3 +150,93 @@ class WriteComment extends React.Component {
 
 export default connect(mapStateToProps, mapDispatchToProps)(WriteComment);
 ```
+
+### Cache Controlling
+```js
+import { schema } from 'normalizr';
+
+const BASE_URL = process.env.REACT_APP_API_BASE || '';
+const API_URL = BASE_URL;
+const cacheDuration = 60000;
+
+// define normalizr response schemas
+const searchTermSchema = new schema.Entity('SearchTerm');
+
+export default {
+    getSearchSuggestions: {
+        url: `${API_URL}/search/suggestions/?q=:searchTerm`,
+        method: 'GET',
+        cacheDuration,
+        searchTermSchema
+    }
+}
+```
+### After Success
+```js
+getCurrentUser: {
+    url: `${API_URL}/settings`,
+    method: 'GET',
+    afterSuccess: (request: any, dispatch: any, getState: any) => {
+        const data: IGetSettingsResponse = getResultData(getState().apiData, 'getSettings');
+        dispatch(update('app.currentUser', data.user.id));
+    }
+},
+```
+### Purge API Data (for deleting local storage)
```

Comment:
Ik denk dat de example hier meer is voor gewoon het verwijderen van api data uit je state voor bijvoorbeeld een logout. Maak er maar de logout case expliciet van, misschien "Removing api data from the store" als titel met een logout functie zoals nu in het voorbeeld.
Je kan evt nog een apart voorbeeld maken van "Using with redux-persist" als dat gebruik niet voor de hand liggend is. Ik snap dat purge vooral belangrijk is met persist, maar ook zonder persist zou je dit moeten doen bij een logout.